Vagrantfile

```diff
@@ -63,6 +63,8 @@ Vagrant.configure(2) do |config|
     set -xe
     sed -i -e "s/PasswordAuthentication no/PasswordAuthentication yes/" /etc/ssh/sshd_config
     systemctl restart sshd
+    # Disable swap
+    sed -i '/ swap / s/^/#/' /etc/fstab
```

Comment:
Can we move that to the provision script?

Reply:
Sure